DuckDB.NET.Data/DuckDBConnectionStringBuilder.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using DuckDB.NET.Data.ConnectionString;
+
+namespace DuckDB.NET.Data
+{
+    public class DuckDBConnectionStringBuilder : IDuckDBConnectionString
+    {
+        public const string InMemory = ":memory:";
```

Comment:
What if instead of this we have an entire connection string for the in-memory database? Should that belong to `DuckDBConnectionString` instead?